### ISSUE TITLE
Motif embeddings

### ISSUE THREAD

**Issue by HiokHian** [microsoft/molecule-generation#41]
Hi,

It is mentioned in the paper that motif embeddings of size 64 were learned and concatenated to the atom input. Could I check, how do we generate these learned embeddings? Or are they already included in the partial node features when we run the preprocessing command using the CLI? 

Additionally what does partial_node_categorical_features refer to in the input data? Thanks!

**Comment by kmaziarz** [comment]
During preprocessing, nodes that are part of motifs are annotated with discrete identifiers of the motifs they are part of (called "categorical features" in the code). These are not yet the motif _embeddings_ - as you noted these are learned, and so are part of the weights of a trained model. In the forward pass, the categorical features are mapped to motif embeddings through an embedding lookup [here](https://github.com/microsoft/molecule-generation/blob/cbb366035acf5c010c63158ec4a73ed48321084b/molecule_generation/models/moler_vae.py#L217).

Whenever we say "partial" in the code, we mean the partial graphs the decoder learns to extend. This is in contrast with the full graphs that are mapped to the latent space by the encoder. Therefore, `partial_node_categorical_features` denotes the motif ids detected in the different partial graphs.

If you want to extract the motif embeddings that were learned by a particular model, then take a look at the call to `tf.nn.embedding_lookup` in the code snippet I linked above. Of course for that you'll need a trained MoLeR checkpoint, such as the one we linked in the README.
